saleor/menu/utils.py

```diff
@@ -1,7 +1,6 @@
-from django.db import transaction
-
 from ..menu.models import Menu
 
```

Comment:
Remove new line.

Reply:
This was not altered. Should the line be removed anyway?